### ISSUE TITLE
Content jumps when presenting nested ideal presentation

### ISSUE THREAD

**Issue by PhilipDukhov** [nathantannar4/Transmission#22]
I have second ideal presentation presented over the first one.

When the second one appears, first one moves down, so user can only see one at the time - which looks super cool.

But content of the second view jumps up without animation when animation starts. When second one is dismissed, first one appears with nice expected animation.

Below stop frame displays view state when animation starts: 

<img src="https://github.com/nathantannar4/Transmission/assets/6103621/9088d5ac-6029-44ab-acb7-e37b0a16c9cc" width="300" />

```swift
@State var isPresented = false

var body: some View {
    ZStack {}
        .presentation(
            transition: .sheet(
                options: .init(
                    detents: [.ideal],
                    prefersSourceViewAlignment: true,
                    options: .init()
                )
            ),
            isPresented: .constant(true)
        ) {
            Button {
                withAnimation {
                    isPresented = true
                }
            } label: {
                Image(systemName: "photo.tv")
                    .resizable()
                    .frame(maxWidth: .infinity)
                    .aspectRatio(contentMode: .fill)
            }
            .frame(maxHeight: .infinity, alignment: .top)
                .presentation(
                    transition: .sheet(
                        options: .init(
                            detents: [.ideal],
                            options: .init(
                                preferredPresentationBackgroundColor: .red
                            )
                        )
                    ),
                    isPresented: $isPresented
                ) {
                    Text("title")
                }
        }
}
```
<img src="https://github.com/nathantannar4/Transmission/assets/6103621/fc0432f5-576e-4142-8879-28928ab8e86b" width="300" />

**Comment by nathantannar4** [comment]
Will look into seeing if I can improve this by default!

**Comment by nathantannar4** [comment]
Will be fixed in 1.1.0
